package.json

```diff
@@ -26,7 +26,7 @@
     "@emurgo/cip14-js": "^2.0.0",
     "@fontsource/ubuntu": "^4.4.5",
     "@hot-loader/react-dom": "^17.0.1",
-    "@ledgerhq/hw-transport-webusb": "^6.11.2",
+    "@ledgerhq/hw-transport-webusb": "6.11.2",
```

Comment:
It was necessary to pin this dependency to the old version because the newest version broke the test suite